angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -299,6 +302,8 @@ namespace :db_connector do
       "LBLVIO-LOW PRESSURE BOILER",
       "LL1081-LOCAL LAW 10/81 - ELEVATOR",
       "LL6291-LOCAL LAW 62/91 - BOILERS",
+      "C-CONSTRUCTION",
+      "B-BOILER",
     ]
```

Comment:
Yup, good call. Since we're working within Rails, I moved this list into the model.